src/main/java/core/basesyntax/classes/functional/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.classes.functional;
+

```

Comment:
Avoid starting the enum implementation with an empty line. Remove this redundant empty line as per the checklist guidelines.